### ISSUE TITLE
Add repology badge to README

### ISSUE THREAD

**Issue by luzpaz** [Canop/dysk#60]
[![Packaging status](https://repology.org/badge/tiny-repos/dysk.svg)](https://repology.org/project/dysk/versions)  
Helps to track what downstream package repos have dysk and it respective version.

**Comment by Canop** [comment]
Good idea. I didn't know they had a badge. It's less cumbersome than their usual list
